e2e/tests/services.rs

```diff
@@ -161,12 +162,16 @@ impl OrderbookServices {
             Duration::from_secs(120),
         ));
 
+        let registry = Registry::default();
+        let metrics = Arc::new(Metrics::new(&registry).unwrap());
         orderbook::serve_task(
             db.clone(),
             orderbook.clone(),
             fee_calculator,
             price_estimator.clone(),
             API_HOST[7..].parse().expect("Couldn't parse API address"),
+            registry,
+            metrics,
```

Comment:
Oh I see that we had to include it in the e2e test because we want to serve the orderbook which requires these metrics. Should we consider making these optional and not send them along with e2e testing?

Reply:
See Felix' comment about NoopMetrics  above.